docs/working-with-velero.md

```diff
@@ -1,25 +1,23 @@
 # Integrate Velero to back up and restore Karmada resources
 
-[Velero](https://github.com/vmware-tanzu/velero) gives you tools to back up and restore 
-your Kubernetes cluster resources and persistent volumes. You can run Velero with a public
-cloud platform or on-premises.
+[Velero](https://github.com/vmware-tanzu/velero) gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes. You can run Velero with a public cloud platform or on-premises.
```

Comment:
I don't think we need to put the content in one line.

Reply:
OK, I have modified it.